src/crypto/index.js

```diff
@@ -131,6 +131,23 @@ export function keyPair () {
   }
 }
 
+/**
+ * Returns a new signing key pair generated deterministically from a 32-byte seed
+ * @return {Object}
+ *  publicKey {string}
+ *  secretKey {string}
+ */
+export function fromSeed (seed) {
+  const pair = nacl.sign.keyPair.fromSeed(seed)
```

Comment:
If pass `undefined`, it will return new pair?

Reply:
it will throw, I've extended the test coverage explaining the cases